hawc/apps/assessment/forms.py

```diff
@@ -245,6 +240,18 @@ def clean(self):
             if not cleaned_data.get("uncertainty"):
                 msg = "Required for Noncancer evaluation types."
                 self.add_error("uncertainty", msg)
+        if (
```

Comment:
added an additional data-check and a test - minor scope creep for this PR, but it was quicker to add than write another card :).